packages/neon/neon/lib/src/sort_box/sort_box_builder.dart

```diff
@@ -2,26 +2,43 @@ import 'package:flutter/widgets.dart';
 import 'package:neon/src/settings/models/select_option.dart';
 import 'package:sort_box/sort_box.dart';
 
+/// Signature for a function that creates a widget for a given sorted list.
+///
+/// Used by [SortBoxBuilder] to display a sroted list of items.
+typedef SortBoxWidgetBuilder<R> = Widget Function(BuildContext context, List<R> sorted);
```

Comment:
```suggestion
typedef SortBoxWidgetBuilder<T> = Widget Function(BuildContext context, List<T> sorted);
```

I usually use T as the first generic type, but of course you can choose whatever you want. I can see that it might more sense as later on the generic R is passed to it